app/views/pages/_badges.html.erb

```diff
@@ -0,0 +1,30 @@
+<h2 class="govuk-heading-m">
+  Your 2020 to 2021 trainees
+</h2>
+
+<div class="govuk-grid-row govuk-!-margin-bottom-6">
```

Comment:
There's a lot of code repetition here, how do you feel about:

```
<div class="govuk-grid-row govuk-!-margin-bottom-6">
  <% %i[draft submitted_for_trn trn_received].map do |state| %>
    <div class="govuk-grid-column-one-third">
      <%= render TraineeStatusCard::View.new(state: state, target: trainees_path("state[]": state), user: current_user) %>
    </div>
  <% end %>
</div>
```

Reply:
Definitely better! DRY and all that